### PR TITLE
[NCL-4254] added field for DTO and moved a const

### PR DESCRIPTION
**Pull request by michalovjan**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

This change is needed to be merged before the Causeway part!

### CODE REVIEW

**Comment by michalovjan** [comment]
@alexcreasy could you look at the naming of the new field in Build DTO. Is this ok or should we use better naming? That field signifies the URL used to clone git repo when building.

File: constants/src/main/java/org/jboss/pnc/constants/Patterns.java [hunk 1]
```diff
@@ -53,5 +53,10 @@
      * the repository URL.
      */
     public static final String INTERNAL_REPOSITORY_NAME = "(\\/[\\w\\.:\\~_-]+)+(\\.git)(?:\\/?|\\#[\\d\\w\\.\\-_]+?)$";
-    
+
+    /**
+     * Represents an key in attribute of {@link org.jboss.pnc.model.ProductVersion}. It's value is
+     * tag prefix in Brew for a Version.
+     */
+    public static final String ATTRIBUTE_KEY_BREW_TAG_PREFIX = "BREW_TAG_PREFIX";
```
Comment (janinko) [inline constants/src/main/java/org/jboss/pnc/constants/Patterns.java]: This is not a regex pattern, move it to an appropriate class.

Comment (michalovjan) [reply constants/src/main/java/org/jboss/pnc/constants/Patterns.java]: Nice catch! Will do.

**Comment by michalovjan** [comment]
retest this please

**Pull request merged**